src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/server/ProxyCacheIT.java

```diff
@@ -210,32 +364,28 @@ public void shouldExpireSMaxage() throws Exception
         k3po.finish();
     }
 
-    // TODO expires headers
-    // TODO quoted maxage header
-    // TODO quoted smaxage header
-
     @Test
     @Specification({
         "${route}/proxy/controller",
-        "${streams}/cache.by.default.for.5.seconds/accept/client",
-        "${streams}/cache.by.default.for.5.seconds/connect/server",
+        "${streams}/cache.by.default.for.0.seconds/accept/client",
+        "${streams}/cache.by.default.for.0.seconds/connect/server",
     })
-    public void shouldCacheDefaultCacheableFor5Seconds() throws Exception
+    public void shouldCacheDefaultCacheableFor0Seconds() throws Exception
     {
         k3po.finish();
     }
 
     @Test
     @Specification({
         "${route}/proxy/controller",
-        "${streams}/expire.cache.by.default.for.5.seconds/accept/client",
-        "${streams}/expire.cache.by.default.for.5.seconds/connect/server",
+        "${streams}/expire.cache.by.default.for.0.seconds/accept/client",
+        "${streams}/expire.cache.by.default.for.0.seconds/connect/server",
```

Comment:
I couldn't find this script from spec repo and change name of the test case.

Reply:
this is fixed.